bin/dotd

```diff
@@ -68,13 +68,19 @@ def open_in_default_browser(url)
   end
 end
 
+def run_on(server_name, command)
+  # escape any double quotes in the command string
+  command = %Q["#{command}"]
+  system "ssh -t gateway.code.org ssh -t #{server_name} \"#{command}\""
```

Comment:
Ahh, I forgot that we're going up through _two_ levels of server, so we actually need that many levels of escaping; as it turns out, we weren't _actually_ wrapping `user_name` in quotes in our original call to `content-push`; giving it a name with whitespace would cause it to blow up.

No longer!